test/utils.ts

```diff
@@ -49,14 +49,16 @@ export function assertValidSchema(
          * @default {strict:false}
          */
         ajvOptions?: AjvOptions;
-    }
+    },
+    discriminatorType?: Config["discriminatorType"]
```

Comment:
We should use an object for all these arguments now. This would remove the need for having so many undefined. Can you do that in a follow up pull request?

Reply:
hi @domoritz yes, i was thinking the same thing... sure i'll submit one. thanks for merging